src/java.base/share/classes/sun/security/ssl/SignatureScheme.java

```diff
@@ -62,6 +68,12 @@ enum SignatureScheme {
                                     "EC",
                                     NamedGroup.SECP521_R1,
                                     ProtocolVersion.PROTOCOLS_TO_13),
+    // Brainpool signature defintion for curve ecdsa_brainpoolP512r1tls13_sha512 as per RFC 8734.
+    ECDSA_BRAINPOOLP512R1TLS13_SHA512  (0x081C, "ecdsa_brainpoolP512r1tls13_sha512",
```

Comment:
Suggest removing the spaces before `(` (others literals appear to attempt to line up the argument list when the name is short - not the case here).